### ISSUE TITLE
[1.12.4]Can't resolve 'linqts'

### ISSUE THREAD

**Issue by Zonciu** [kutyel/linq.ts#161]
`dist` folder structure changed but `package.json` doesn't modified.
dist:
```
index.ts
index.d.ts
-- src
    |-- ...
```

package.json
```
  "main": "dist/linqts.js",
  "module": "dist/linqts.es.js",
  "types": "dist/index.d.ts",
  "umd:main": "dist/linqts.umd.js",
```

**Comment by kutyel** [comment]
Where is the error? I opened the dist folder and see exactly the same structure, how did you run into the error?

**Comment by Zonciu** [comment]
I'd tried both Vue and React:

Vue
[vue.zip](https://github.com/kutyel/linq.ts/files/3310386/vue.zip)
```
 ERROR  Failed to compile with 1 errors                                                                         20:30:38

This dependency was not found:

* linqts in ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/App.vue?vue&type=script&lang=js&

To install it, you can run: npm install --save linqts
```

React
[react.zip](https://github.com/kutyel/linq.ts/files/3310387/react.zip)
```
Failed to compile.

./src/App.js
Module not found: Can't resolve 'linqts' in 'E:\dev\zonciu\linqtest\linqtest\src'
```

Rename ` "main": "dist/linqts.js",` to `"main": "dist/index.js",` in package.json then the project works.

**Comment by kutyel** [comment]
@Zonciu ok, I've published a fixed version 1.12.5, can you try if that works? 😉

**Comment by Zonciu** [comment]
Yes it works.👌
